crates/shim-sgx/src/handler/mod.rs

```diff
@@ -199,58 +301,13 @@ impl guest::Handler for Handler<'_> {
     fn mprotect(
         &mut self,
         _platform: &impl Platform,
-        addr_in: NonNull<c_void>,
-        length_in: c_size_t,
+        addr: NonNull<c_void>,
+        length: c_size_t,
```

Comment:
```suggestion
        len: c_size_t,
```
https://linux.die.net/man/2/mprotect

Reply:
fixed